.github/stale.yml

```diff
@@ -0,0 +1,18 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 30
+# Number of days of inactivity before a stale issue is closed
```

Comment:
Is this _issues_ or _PRs_? (Or both?) 

I'm not sure if we should be closing issues, but I agree we should be closing PRs

Reply:
Both, I agree w/ you on issues, but expect it will force us to comment.

Reply:
I think I have a few changes to this.  Hold off for a bit.

Reply:
PTAL